packages/ui/src/bounty/modals/SubmitJudgementModal/SubmitJudgementModal.tsx

```diff
@@ -93,28 +95,36 @@ export const SubmitJudgementModal = () => {
       const winnersApi = validWinners.map(
         ({ winner, reward }) =>
           [
-            createType('u32', Number(bounty.entries?.find((entry) => entry.worker.id === winner.id)?.id) ?? 0),
-            { Winner: { reward: createType('u128', reward) } },
+            createType<EntryId, 'EntryId'>(
+              'EntryId',
+              Number(bounty.entries?.find((entry) => entry.worker.id === winner.id)?.id) ?? 0
+            ),
+            createType<OracleWorkEntryJudgment, 'OracleWorkEntryJudgment'>('OracleWorkEntryJudgment', {
+              Winner: { reward: createType('u128', reward) },
+            }),
           ] as const
       )
 
-      // todo handle rejected when submitOracleJudgement is ready
-      // const validRejections = state.context.rejected.filter(
-      //   (rejection) => rejection.rejected
-      // ) as Required<BountyRejected>[]
-      // const rejectedApi = validRejections.map(
-      //   (loser) =>
-      //     [
-      //       createType('u32', Number(bounty.entries?.find((entry) => entry.worker.id === loser.rejected.id)?.id) ?? 0),
-      //       null,
-      //     ] as const
-      // )
+      const validRejections = state.context.rejected.filter(
+        (rejection) => rejection.rejected
+      ) as Required<BountyRejected>[]
+      const rejectedApi = validRejections.map(
+        (loser) =>
+          [
+            createType('u32', Number(bounty.entries?.find((entry) => entry.worker.id === loser.rejected.id)?.id) ?? 0),
```

Comment:
Just to be consistent with the winner type:
```suggestion
            createType<EntryId, 'EntryId'>(
              'EntryId',
              Number(bounty.entries?.find((entry) => entry.worker.id === loser.rejected.id)?.id) ?? 0
            ),
```